sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/CosmosStoredProcedure.java

```diff
@@ -14,20 +14,16 @@
  */
 public class CosmosStoredProcedure {
     private final String id;
-    private final CosmosContainer container;
     private final CosmosAsyncStoredProcedure storedProcedure;
 
     /**
      * Instantiates a new Cosmos sync stored procedure.
      *
      * @param id the id
-     * @param container the container
      * @param storedProcedure the stored procedure
      */
-    public CosmosStoredProcedure(String id, CosmosContainer container, CosmosAsyncStoredProcedure storedProcedure) {
-
+    public CosmosStoredProcedure(String id, CosmosAsyncStoredProcedure storedProcedure) {
```

Comment:
This passes async version to the constructor of the sync version. This should be required only by implementation not public api right?

I think the constructor should be package private.

Reply:
Changed to package-private.